packages/wearable_rotary/tizen/src/log.h

```diff
@@ -0,0 +1,25 @@
+#ifndef __LOG_H__
+#define __LOG_H__
+
+#include <dlog.h>
+
+#ifdef LOG_TAG
+#undef LOG_TAG
+#endif
+#define LOG_TAG "WearableRotaryPlugin"
+
+#ifndef __MODULE__
+#define __MODULE__ \
+  (strrchr(__FILE__, '/') ? strrchr(__FILE__, '/') + 1 : __FILE__)
```

Comment:
Please update this according to the new form: https://github.com/flutter-tizen/flutter-tizen/pull/123